apps/fz_http/lib/fz_http/api_tokens/api_token.ex

```diff
@@ -0,0 +1,51 @@
+defmodule FzHttp.ApiTokens.ApiToken do
+  @moduledoc """
+  Stores API Token metadata to check for revocation.
+  """
+  use FzHttp, :schema
+  import Ecto.Changeset
+
+  @max_per_user 10
+
+  schema "api_tokens" do
+    field :expires_at, :utc_datetime_usec
+
+    # User-friendly way to set expires_at
+    field :expires_in, :integer, virtual: true, default: 30
```

Comment:
We don't know the unit used here, maybe it's better to add units to the field name:
```suggestion
    field :expires_in_days, :integer, virtual: true, default: 30
```

Reply:
👍

Reply:
I've removed this field altogether in response to https://github.com/firezone/firezone/pull/1155#discussion_r1057397968